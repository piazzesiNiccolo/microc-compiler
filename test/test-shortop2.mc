int main(){
    int a = 2;
    float b =2.5;

    print(a+=1);
    print(a-=1);
    print(a*=4);
    print(a/=2);
    print(a%=4);

    printfloat(b+=1.);
    printfloat(b-=1.);
    printfloat(b*=2.);
    printfloat(b/=3.);
    


}