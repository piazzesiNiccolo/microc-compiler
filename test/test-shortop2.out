3
2
8
4
0
3.500000
2.500000
5.000000
1.666667
